sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_transport.py

```diff
@@ -716,7 +716,11 @@ def connect(self):
                 WebSocketTimeoutException,
                 WebSocketConnectionClosedException
             )
-
+        except ImportError as exc:
+            raise ValueError(
```

Comment:
Should raise an ImportError:
https://github.com/Azure/azure-sdk-for-python/blob/3e3056c26ae8ce4484142fcb9901e1334d54a6fe/sdk/core/azure-core/azure/core/pipeline/transport/__init__.py#L98

Reply:
Should this be surfaced as an ImportError as well? Currently, we wrap this ValueError with EventHubError. But can be changed so that the ImportError doesn't get wrapped.

Reply:
surfacing ImportError not EHError.